docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,238 @@
+If you rely on the HA capability, Knative operators are currently not helping with that.
+
+The open source versions of Knative operators have not been tested on any existing production environment. Please use
+them for development or test purpose. However, we encourage you to experiment Knative operators on top of the Kubernetes
+cluster for your production environment, helping us find out what we can potentially improve in Knative operators by
+reporting your issues.
+
+## Prerequisites:
+
+Knative Serving needs an ingress or gateway to route inbound network traffic to the services. There are multiple options
+that can be used as the ingress candidates: Istio, Ambassador, Contour, Gloo, Kourier, etc. However, to install Knative
+with operators, we only support Istio as the ingress for now. Knative Operators are currently working on enabling the
+support for more ingress options. Since we talk about the installation on generic Kubernetes service, Istio can be
+installed with the following steps:
+
+1. [Download and install Istio](https://knative.dev/development/install/installing-istio/#downloading-istio-and-installing-crds). Go through all the 4 sub-steps.
+2. [Update your Istio to use cluster local gateway](https://knative.dev/development/install/installing-istio/#updating-your-install-to-use-cluster-local-gateway).
+
+## Install Knative Serving with Operator:
+
+All the releases for Knative Serving Operator can be found on [this page](https://github.com/knative/serving-operator/releases).
+
+### From releases:
+
+Replace \<version\> with the latest version or the version you would like to install, and run the following command to
+install Knative Serving Operator:
+
+```
+kubectl apply -f https://github.com/knative/serving-operator/releases/download/<version>/serving-operator.yaml
+```
+
+### From source code:
+
+The source code is available on this page. You need to install the building tool [ko](https://github.com/google/ko) first. After downloading the
+source code, you can install the operator in the root directory of the source code with the following command:
+
+```
+ko apply -f config/
+```
+
+Verify the installation of Knative Serving Operator with the command:
+
+```
+kubectl get deployment knative-serving-operator
+```
+
+You should see the deployment is ready, if the operator is installed.
+
+Use the following command to track the log of the operator:
+
+```
+kubectl logs -f $(kubectl get pods -l name=knative-serving-operator -o name)
+```
+
+If everything goes fine, it is time to install Knative Serving by installing the custom resource of Knative Serving
+operator. Operator supports to install Knative Serving under any namespace, which needs to be created as well. Then,
+we can create a custom resource with empty spec section. Technically, you can use any namespace, but we recommend you
+to use knative-serving. In the rest of this tutorial, we keep on using knative-servingas the namespace to create the
```

Comment:
typo: change knative-servingas to knative-serving as

Reply:
done.